open-cluster-management/backlog#14867 skipMachinePools

cadawson opened the pull request.

cadawson committed:
open-cluster-management/backlog#14867 skipMachinePools

cadawson committed:
open-cluster-management/backlog#14867 Add space

dhrpatel4 reviewed (commented):
Hi this looks good, but Now we will have to modify line 73
so I believe that if we add this spec then that SNO cluster pool creating extra worker nodes will not exists any more
1:27
so if user create sno cluster pool without spec then they will get those extra machine pools but if they add spec from yaml then no extra nodes will be created

@cadawson Just adding this based on our discussion

cadawson committed:
open-cluster-management/backlog#14867 rem create clusters ki

Removing the other known issue about the clusters being created for SNO clusters. By adding the flag in this one, it resolves the other one.

dhrpatel4 reviewed (approved):
Thanks @cadawson for the prompt changes.
looks good to me
